spec/api_maker/create_command_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+describe ApiMaker::CreateCommand do
+  let(:project) { create :project }
+  let(:user) { create :user }
+
+  let(:ability) { ApiMaker::Ability.new(args: {current_user: user}) }
+  let(:api_maker_args) { {current_user: user} }
+  let(:collection) { Task.accessible_by(ability) }
+  let(:controller) { instance_double("ApplicationController", api_maker_args: api_maker_args, current_ability: ability, current_user: user) }
+  let(:helper) do
+    ApiMaker::CommandSpecHelper.new(
+      collection: collection,
+      command: ApiMaker::CreateCommand,
+      controller: controller
+    )
+  end
+
+  it "returns validation errors" do
+    task_params = {
+      name: "Test task",
+      user_id: user.id
+    }
+
+    command = helper.add_command(args: {save: {task: task_params}})
+    helper.execute!
+
+    expect(command.result).to eq(
```

Comment:
What happens if you use `as_json`?
```suggestion
    expect(command.result.as_json).to eq(
```

Reply:
Is working with `.as_json` does just not look as nice 😄